plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -512,7 +544,22 @@ static obs_properties_t *nvenc_properties(void *unused)
 	obs_properties_add_int(props, "gpu", obs_module_text("GPU"), 0, 8, 1);
 
 	obs_properties_add_int(props, "bf", obs_module_text("BFrames"),
-			0, 4, 1);
+		0, 4, 1);
+	obs_properties_add_bool(props, "spatialaq",
+		obs_module_text("NVENC.SpatialAQ"));
+	obs_properties_add_bool(props, "temporalaq",
+		obs_module_text("NVENC.TemporalAQ"));
+	obs_properties_add_bool(props, "forcedidr",
+		obs_module_text("NVENC.forcedidr"));
+	obs_properties_add_int(props, "spatialaqstrength",
+		obs_module_text("NVENC.SpatialStrength"),
+		1, 15, 1);
+	obs_properties_add_int(props, "rclookahead",
+		obs_module_text("NVENC.rclookahead"),
+		0, 32, 1);
+	obs_properties_add_int(props, "surfaces",
+		obs_module_text("NVENC.surfaces"),
+		32, 64, 1);
```

Comment:
Style:  This line can safely be at the end of the preceding line without exceeding the column limit.